src/azul/queues.py

```diff
@@ -430,19 +433,26 @@ def manage_lambdas(self, queues: Mapping[str, Queue], enable: bool):
             def submit(f, *args, **kwargs):
                 futures.append(tpe.submit(f, *args, **kwargs))
 
+            indexer = load_app_module('indexer')
+            lambdas = [
```

Comment:
Any way we can deduce this by checking which lambdas have a `.queue` attribute rather than hard-coding this list?